File: shared/src/main/java/xyz/klinker/messenger/shared/view/AlwaysOffSwitchPreference.java
package xyz.klinker.messenger.shared.view;

import android.content.Context;
import android.support.v14.preference.SwitchPreference;
import android.util.AttributeSet;

import xyz.klinker.messenger.shared.R;

public class AlwaysOffSwitchPreference extends SwitchPreference {

    public AlwaysOffSwitchPreference(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        init();
    }

    public AlwaysOffSwitchPreference(Context context, AttributeSet attrs) {
        super(context, attrs);
        init();
    }

    public AlwaysOffSwitchPreference(Context context) {
        super(context);
        init();
    }

    private void init() {
        setChecked(false);
    }
}

File: shared/src/main/java/xyz/klinker/messenger/shared/view/AlwaysOffSwitchPreference.kt
package xyz.klinker.messenger.shared.view

import android.content.Context
import android.util.AttributeSet
import androidx.preference.SwitchPreference

import xyz.klinker.messenger.shared.R

class AlwaysOffSwitchPreference : SwitchPreference {

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr)
    constructor(context: Context, attrs: AttributeSet) : super(context, attrs)
    constructor(context: Context) : super(context)

    init {
        isChecked = false
    }
}
